docs/source/format/driver_manifests.rst

```diff
@@ -335,33 +341,40 @@ to control which directories will be searched for manifests, with the behavior b
        the type ``adbc_core::driver_manager::LoadFlags``, which has the following constants:
-       * `LOAD_FLAG_SEARCH_ENV` - search the environment variable ``ADBC_CONFIG_PATH``
-       * `LOAD_FLAG_SEARCH_USER` - search the user configuration directory
-       * `LOAD_FLAG_SEARCH_SYSTEM` - search the system configuration directory
-       * `LOAD_FLAG_ALLOW_RELATIVE_PATHS` - allow a relative path to be used
-       * `LOAD_FLAG_DEFAULT` - default value with all flags set
 
-For unix-like platforms, (e.g. Linux, macOS), the driver manager will search the following directories based on the options provided, in
+       * ``LOAD_FLAG_SEARCH_ENV`` - search the environment variable ``ADBC_CONFIG_PATH``
+       * ``LOAD_FLAG_SEARCH_USER`` - search the user configuration directory
+       * ``LOAD_FLAG_SEARCH_SYSTEM`` - search the system configuration directory
+       * ``LOAD_FLAG_ALLOW_RELATIVE_PATHS`` - allow a relative path to be used
+       * ``LOAD_FLAG_DEFAULT`` - default value with all flags set
+
+Unix-like Platforms
+^^^^^^^^^^^^^^^^^^^
+
+For Unix-like platforms, (e.g. Linux, macOS), the driver manager will search the following directories based on the options provided, in
 the given order:
 
 #. If the ``LOAD_FLAG_SEARCH_ENV`` load option is set, then the environment variable ``ADBC_CONFIG_PATH`` will be searched
 
-   * ``ADBC_CONFIG_PATH`` is a colon-separated list of directories to search for ``${name}.toml``
+   ``ADBC_CONFIG_PATH`` is a colon-separated list of directories to search for ``${name}.toml``
 
 #. If the ``LOAD_FLAG_SEARCH_USER`` load option is set, then a user-level configuration directory will be searched
 
    * On macOS, this will be ``~/Library/Application Support/ADBC``
-   * On Linux (and other unix-like platforms), the ``XDG_CONFIG_HOME`` environment variable is checked first. If it is set, the driver manager
+   * On Linux (and other Unix-like platforms), the ``XDG_CONFIG_HOME`` environment variable is checked first. If it is set, the driver manager
      will search ``$XDG_CONFIG_HOME/adbc``, otherwise it will search ``~/.config/adbc``
 
 #. If the ``LOAD_FLAG_SEARCH_SYSTEM`` load option is set, the driver manager will search ``/etc/adbc`` if it exists
 
+Windows
+^^^^^^^
+
 Things are slightly different on Windows, where the driver manager will also search for driver information in the registry just as
-would happen for ODBC drivers. The search for a manifest on windows would be the following:
+would happen for ODBC drivers. The search for a manifest on Windows would be the following:
 
 #. If the ``LOAD_FLAG_SEARCH_ENV`` load option is set, then the environment variable ``ADBC_CONFIG_PATH`` will be searched
 
-    * ``ADBC_CONFIG_PATH`` is a semicolon-separated list of directories to search for ``${name}.toml``
+   ``ADBC_CONFIG_PATH`` is a semicolon-separated list of directories to search for ``${name}.toml``
```

Comment:
Same here, the * should stay, but remove the leading space so that the * is lined up with the `I` of `If`

Reply:
Done in 0a68b5154d116b1bf8f572aa98d85790b47a7db3.